src/tls/README.md

```diff
@@ -9,10 +9,9 @@ multiplexes sessions of different users in one process. Thus any vulnerability
 can easily lead to complete privilege escalation across all running and future
 sessions.
 
-Right now this does not have any extra features over using cockpit-ws directly
-(other than better isolation). In the future, it will be able to do certificate
-based client authentication (smart cards, browser-imported certificates, and
-similar), which greatly aggravates the above security trust issue.
+That separation permits Cockpit to do certificate based client authentication
+(smart cards, browser-imported certificates, and similar) in a sufficiently
+robust and trustworthy manner.
```

Comment:
I feel like this is a bit too hand-wavy.  I'd specifically mention that using cockpit-tls allows us to isolate the main parts of cockpit on a per-client-certificate basis.

Reply:
Right, I reworked the text.